src/applications/caregivers/tests/components/FormFields/AddressWithAutofill.unit.spec.js

```diff
@@ -4,116 +4,114 @@ import { render } from '@testing-library/react';
+      formData: {
+        street: '1350 I St. NW',
+        street2: 'Suite 550',
+        city: 'Washington',
+        state: 'DC',
+        postalCode: '20005',
+        county: 'Arlington',
+        'view:autofill': autofill,
+      },
+      errorSchema: {
+        city: errorSchemas.required,
+        county: errorSchemas.required,
+        postalCode: errorSchemas.required,
+        state: errorSchemas.required,
+        street: errorSchemas.required,
+        street2: errorSchemas.empty,
+        'view:autofill': errorSchemas.empty,
+      },
+      idSchema: {
+        $id: 'root_caregiverAddress',
+        city: { $id: 'root_caregiverAddress_city' },
+        county: { $id: 'root_caregiverAddress_county' },
+        postalCode: { $id: 'root_caregiverAddress_postalCode' },
+        state: { $id: 'root_caregiverAddress_state' },
+        street: { $id: 'root_caregiverAddress_street' },
+        street2: { $id: 'root_caregiverAddress_street2' },
+        'view:autofill': { $id: 'root_caregiverAddress_view:autofill' },
+      },
+      schema: addressWithAutofillSchema(address),
+      onChange: () => {},
     },
-    idSchema: {
-      $id: 'root_primaryAddress',
-      city: { $id: 'root_primaryAddress_city' },
-      postalCode: { $id: 'root_primaryAddress_postalCode' },
-      state: { $id: 'root_primaryAddress_state' },
-      street: { $id: 'root_primaryAddress_street' },
-      street2: { $id: 'root_primaryAddress_street2' },
-      'view:autofill': { $id: 'root_primaryAddress_view:autofill' },
+    mockStore: {
+      getState: () => ({
+        form: {
+          data: {
+            veteranAddress: {
+              street: '1350 I St. NW',
+              street2: 'Suite 550',
+              city: 'Washington',
+              state: 'DC',
+              postalCode: '20005',
+            },
+          },
+        },
+      }),
+      subscribe: () => {},
+      dispatch: () => {},
     },
-    schema: address,
-    onChange: () => {},
-  };
+  });
+
+  context('when not in review mode', () => {
+    const { mockStore, props } = getData({});
 
-  describe('when review mode is `false`', () => {
     it('should render the form', () => {
-      const view = render(
+      const { container } = render(
         <Provider store={mockStore}>
-          <AddressWithAutofill {...defaultProps} />
+          <AddressWithAutofill {...props} />
         </Provider>,
       );
+      const expectedFieldTypes = 'va-checkbox, va-text-input, va-select';
       const selectors = {
-        fieldset: view.container.querySelector('.cg-address-with-autofill'),
-        inputs: view.container.querySelectorAll(
-          'va-checkbox, va-text-input, va-select',
-        ),
+        fieldset: container.querySelector('.cg-address-with-autofill'),
+        inputs: container.querySelectorAll(expectedFieldTypes),
       };
       expect(selectors.fieldset).to.exist;
       expect(selectors.fieldset).to.not.be.empty;
```

Comment:
So, I'm going to expand on all the test coverage this next sprint. We do have gaps in coverage in a lot of places. So that is going to be a separate effort to get those coverage percentages way up. 

I do agree, though... will make a note to test for the label value.